src/config/configDefaults.ts

```diff
@@ -1,10 +1,15 @@
-export const SHOW_MUTATION_ASSESSOR = true;
+export const SHOW_MUTATION_ASSESSOR = false;
 
-export const ANNOTATION_QUERY_FIELDS = [
+export function annotationQueryFields() {
+    let fields = DEFAULT_ANNOTATION_QUERY_FIELDS;
+    SHOW_MUTATION_ASSESSOR && fields.push('mutation_assessor');
```

Comment:
We should avoid this syntax in a regular function and only use it in the JSX context. Let's change this to

```suggestion
    if (SHOW_MUTATION_ASSESSOR) {
        fields.push('mutation_assessor');
    }
```